pakwheels_clone/cars/models.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib import admin
+from django.db import models
+
+
+from users.models import User
+
+
+class CarManufacturer(models.Model):
+    car_make = models.CharField(max_length=50)
+
+
+class CarModel(models.Model):
+    car_model = models.CharField(max_length=150)
+
+
+class CarFeatures(models.Model):
+    feature = models.CharField(max_length=200)
+
+
+class CarImages(models.Model):
+    image = models.ImageField(upload_to="images")
+
+
+class CarDetails(models.Model):
+    fuel_choices = [
+        ('Petrol', 'Petrol'),
+        ('Diesel', 'Diesel'),
+    ]
+    transmission_choices = [
+        ('Manual', 'Manual'),
+        ('Automatic', 'Automatic'),
+    ]
+    assembly_choices = [
+        ('Local','Local'),
+        ('Imported','Imported'),
+    ]
+    title = models.CharField(max_length=100)
+    model = models.ForeignKey(CarModel, on_delete=models.CASCADE)
```

Comment:
read about string references

Reply:
list of field or model properties should be alphabetically sorted and there should be empty line between normal fields and foreign keys etc